backend/package.json

```diff
@@ -51,4 +51,4 @@
   "prisma": {
     "seed": "ts-node ./utils/userContactPlanSeed.ts"
   }
-}
+}
```

Comment:
let's keep the new line at the end of files! this is a standard convention

Reply:
Im not sure why but when I add the extra space and try to save the file its reverting back and deleting the extra line that I add.

Reply:
done!